build_config/lib/src/parse.dart

```diff
@@ -53,8 +53,8 @@ BuildConfig parseFromYaml(
     parseFromMap(packageName, dependencies,
         loadYaml(configYaml) as Map<String, dynamic>);
 
-BuildConfig parseFromMap(String packageName, Iterable<String> dependencies,
-    Map<String, dynamic> config) {
+BuildConfig parseFromMap(String packageName,
+    Iterable<String> packageDependencies, Map<String, dynamic> config) {
```

Comment:
Let's add an assert to validate that this is not null so it won't blow up on the toList call below.

Reply:
Done